backend/api/ir.py

```
@@ -29,12 +29,113 @@
 
+# ─── Bedrock client ───────────────────────────────────────────────────────────
+
+def _get_bedrock_client():
+    api_key = os.environ.get("BEDROCK_API_KEY")
+    region  = os.environ.get("AWS_DEFAULT_REGION", "us-east-1")
+    if api_key:
+        return boto3.client(
+            "bedrock-runtime",
+            region_name=region,
+            aws_access_key_id=api_key,
+            aws_secret_access_key="bedrock-api-key",  # Bedrock API key auth
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "aws_secret_access_key"

(S106)